pkg/cmd/controller/build.go

```diff
@@ -49,6 +50,23 @@ func RunBuildController(ctx *ControllerContext) (bool, error) {
 	proxyCfgInformer := ctx.ConfigInformers.Config().V1().Proxies()
 	imageContentSourcePolicyInformer := ctx.OperatorInformers.Operator().V1alpha1().ImageContentSourcePolicies()
 
+	fg := ctx.OpenshiftControllerConfig.FeatureGates
+	// 		featuregates.NewObserveFeatureFlagsFunc(sets.NewString("BuildCSIVolumes"), sets.String{}, []string{"featureGates"}),
+	foundBuildCSIVolumes := false
+	foundCSIDriverSharedResource := false
+	if fg != nil && len(fg) > 0 {
+		for _, a := range fg {
+			for _, v := range a {
+				switch strings.TrimSpace(v) {
+				case "CSIDriverSharedResource":
+					foundCSIDriverSharedResource = true
```

Comment:
We don't need to check this feature gate - this gate is used to install the shared resource CSI driver on the cluster.

Reply:
updated @adambkaplan PTAL